README.md

```diff
@@ -40,199 +38,107 @@ If you need more information about setting up your development environment Subst
-│  │     ├── sync_types.rs               // Useful sync types
-│  │     └── utils.rs                    // General utilities
-```
+├── bin             // Node and Runtime for the various Substrate chains
+│  └── ...
+├── deployments     // Useful tools for deploying test networks
+│  └──  ...
+├── diagrams        // Pretty pictures of the project architecture
+│  └──  ...
+├── modules         // Substrate Runtime Modules (a.k.a Pallets)
+│  ├── ethereum     // Ethereum PoA Header Sync Module
+│  ├── substrate    // Substrate Based Chain Header Sync Module
+│  ├── message-lane // Cross Chain Message Passing
+│  └──  ...
+├── primitives      // Code shared between modules, runtimes, and relays
+│  └──  ...
+├── relays          // Application for sending headers and messages between chains
+│  └──  ...
+└── scripts         // Useful development and maintenence scripts
+ ```
 
 ## Running the Bridge
 
 To run the Bridge you need to be able to connect the bridge relay node to the RPC interface of nodes
-on each side of the bridge (home & foreign chain). An easy way to build all the required nodes is
-through Docker.
-
-### Local Development Build
+on each side of the bridge (source and target chain).
 
-#### Building
+### Building
 
-First you'll need to build the bridge node and relay. This can be done as follows:
+First you'll need to build the bridge nodes and relay. This can be done as follows:
 
 ```bash
 # In `parity-bridges-common` folder
 cargo build -p rialto-bridge-node
-cargo build -p ethereum-poa-relay
+cargo build -p millau-bridge-node
+cargo build -p substrate-relay
 ```
 
-Next you'll need to clone the following [fork of OpenEthereum](https://github.com/paritytech/openethereum).
-If you're doing development which only involves the Ethereum to Substrate side of the bridge you may
-use the `master` branch. Otherwise you'll need to checkout the `substrate-builtins-stubs` branch.
-
-```bash
-# Should be at the same level as `parity-bridges-common` folder
-git clone https://github.com/paritytech/openethereum.git openethereum
-git fetch
-git checkout substrate-builtins-stubs
-```
-
-If you've checked out the `substrate-builtins-stubs` branch make sure you've cloned the OpenEthereum
-repo at the same level as `parity-bridges-common` since it references the repo.
-
-Next you'll need to build the Ethereum node:
-
-```bash
-# In `openethereum` folder
-cargo build
-```
-
-#### Running
+### Running
 
 To run a simple dev network you'll can use the scripts located in
-[the `scripts` folder](./scripts). Since the relay connects to both the Substrate and Ethereum
-chains it must be run last.
+[the `scripts` folder](./scripts). Since the relay connects to both Substrate chains it must be run
+last.
 
 ```bash
 # In `parity-bridges-common` folder
-./scripts/run-openethereum-node.sh
 ./scripts/run-rialto-bridge-node.sh
-./scripts/run-eth2sub-relay.sh
+./scripts/run-millau-bridge-node.sh
+./scripts/run-millau-to-rialto-relay.sh
```

Comment:
Hi, I am trying to go through this tutorial, running two substrate node with one bridger. However, i can not find `run-millau-to-rialto-relay.sh` in scripts now? Or I am missing something important.

Reply:
Hey! This script doesn't exist yet. I'll ping you when it's added. In the meantime, check out the [Docker based setup](https://github.com/paritytech/parity-bridges-common/tree/master/deployments) which will spin up a full network deployment.

Reply:
Thanks for your reply. I will try the docker way firstly.

Reply:
@AsceticBear I've opened #572 which adds these scripts.